src/main/java/uk/gov/justice/hmpps/prison/service/AdjudicationsService.java

```diff
@@ -333,6 +325,24 @@ public void deleteOicHearing(final Long adjudicationNumber, final long oicHearin
         oicHearingRepository.delete(hearingToDelete);
     }
 
+    @Transactional(readOnly = true)
+    public List<OicHearingResultDto> getOicHearingResults(
```

Comment:
Thought I'd return the list of results and Adj API can simple check for a result.